src/api/teleporter.c

```diff
@@ -736,6 +736,9 @@ static int process_received_tar_gz(struct ftl_conn *api, struct upload_data *dat
 			// i = 3
 			.archive_name = "setupVars.conf",
 			.destination = config.files.setupVars.v.s
+		},{
+			.archive_name = "05-pihole-custom-cname.conf",
```

Comment:
Without testing the branch: does the extraction also work on files in sub-dirs?

![2024-09-10_22-45](https://github.com/user-attachments/assets/4ea3d988-de67-4734-9cc6-7c3388e8282c)

Reply:
Yes - there is nothing like "directories" in tar but everything is in the filename. However, this is obviously wrong here, thanks for the pointer!

```suggestion
			.archive_name = "dnsmasq.d/05-pihole-custom-cname.conf",
```